src/services/sync/strategies/LDAPSyncer.js

```diff
@@ -55,17 +55,27 @@ class LDAPSyncer extends Syncer {
 	}
```

Comment:
for consistency, we could also add logger logic also for processLdapSchools
also to check, this method returns a promise while the other methods don't return anything and call the sendXXX method async

Reply:
we should check that Promise.all for classes and users, but for logging is fine

Reply:
maybe but the Promise.all was releay pland by me as the sync where redesinge.
It's for that the Users are there before add the classes.
In the proces to add Users as studend or teacher to classes the worker wo do the you use LDAP DNs and the school id to find out the user id and add it then to the class.